From 6f09a6de41e7e35779e0755d9e52a4bf79424109 Mon Sep 17 00:00:00 2001
From: albap <albapompeo@gmail.com>
Date: Sun, 31 Jan 2016 10:50:02 -0200
Subject: [PATCH] Remove nonstandard header sys/sysctl.h

This header prevents me from building Julia on musl libc.

```
/home/alba/julia/src/sys.c:16:24: fatal error: sys/sysctl.h: No such file or directory
compilation terminated.
Makefile:94: recipe for target 'sys.o' failed
make[1]: *** [sys.o] Error 1
```

If I remove it from julia/src/sys.c then it works.

Why is sys/sysctl.h needed in first place? I cannot find it in POSIX.
---
 src/sys.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/sys.c b/src/sys.c
index 9a48871..89069a9 100644
--- a/src/sys.c
+++ b/src/sys.c
@@ -13,7 +13,6 @@
 #ifdef _OS_WINDOWS_
 #include <psapi.h>
 #else
-#include <sys/sysctl.h>
 #include <sys/wait.h>
 #include <sys/ptrace.h>
 #include <unistd.h>
